src/components/home/Landing.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+
+const Landing = () => {
+  return (
+    <div className="bg-aisc-gray flex h-[70vh] w-screen flex-col items-center justify-center">
```

Comment:
```suggestion
    <div className="bg-aisc-gray flex h-[70vh] w-full flex-col items-center justify-center">
```
w-full is better to not cause overflow